src/microbit/pxt_rakettoppskytning/pxt_rakettoppskytning.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "PXT: Rakettoppskytning"
+author: Helene Isnes
+language: nb
+---
+
+
+# Introduksjon {.intro}
+Her skal vi først telle ned fra fem, og så lage en animasjon av en rakett som flyr av gårde.
+
+
+# Steg 1: Nedtelling {.activity}
+
+Først skal vi lage nedtelleren. Du vil at tallet 5 skal vises i et sekund, så 4, så 3 osv. Dette kan du få til ved å bruke en kombinasjon av blokkene `vis tall` og `pause` fra `Basis` kategorien.
+
+
+## Sjekkliste {.check}
+
+- [ ] Få micro:biten til å telle ned fra 5. *Husk at 1000 ms er det samme som 1 sekund*	
+
+- [ ] Koden burde nå se slik ut:
+
+![Bilde som viser nedtelling med `pause` og `vis tall` blokker](nedtelling.png)
+
+# Steg 2: Raketten {.activity}
+
+Nå som vi har fått til nedtelingen trenger vi en rakett. Min rakett ser slik ut:
+
+![Bilde som viser raketten vi skal skyte opp](rakett.png)
+
+Du kan designe raketten din akkurat som du vil.
+
+
+## Sjekkliste {.check}
+
+- [ ] For å skape inntrykket av at raketten skytes opp tegner du ett etter ett bilde der raketten har flyttet seg et steg. Bildene du tegner henger du etter koden i steg 1. 
```

Comment:
For å skape et inntrykk av at raketten skytes opp så tegner du nye bilder på skjermen hele tiden slik at det ser ut som om raketten flytter seg litt og litt. Bildene du tenker legger du etter koden i steg *1*.

Reply:
Dette er kun et forslag. Må muligens finjustere litt mer.